rfcs/0025-nix-core-team.md

```diff
@@ -0,0 +1,82 @@
+---
+feature: nix-core-team
+start-date: 2018-01-31
+author: Graham Christensen
+co-authors: Daniel Peebles, Eelco Dolstra, Peter Simons, Shea Levy, Vladimír Čunát
+related-issues:
+---
+
+# Summary
+[summary]: #summary
+
+Create an experimental Nix Core Team to help lead the direction of
+Nix. This RFC may not be perfect, and we don’t have good answers to
+all the possible questions, but let’s try it.
+
+# Motivation
+[motivation]: #motivation
+
+ - Improve visibility in to how the project operates
+ - Distribute the work Eelco has been doing across more people
+ - "Unstuck" pull requests which are sitting idle
+ - Provide a more diverse group of experiences when evaluating changes
+   to core Nix
+
+# Detailed design
+[design]: #detailed-design
+
+## This team will:
+
+ - Evaluate larger features being proposed to Nix
+ - Serve as a second opinion on Nix changes that Eelco doesn't
+   otherwise see the value to
+ - Make road-mapping decisions
+ - Evaluate a change to determine if it is ready for inclusion
+ - Follow up on unreviewed pull requests
+
+The core team will have a GitHub team, a public mailing list, and
+perhaps an IRC channel. The team will comprise long-term, trusted
+community members who have a deep understanding of Nix and the Nix
+ecosystem.
+
+## To start with, the team will be:
+
+ - Daniel Peebles @copumpkin
+ - Eelco Dolstra @edolstra
+ - Peter Simons @peti
+ - Shea Levy @shlevy
+ - Vladimír Čunát @vcunat
```

Comment:
I've received a bunch of questions (mostly in private) about why I'm not on this list. To start with, I'm not sure why I should be on the list: this isn't really in my wheel-house. My role on putting this team together was primarily as a consensus-builder and trusted neutral party. I'm happy to continue helping in that capacity as the team sees fit.

Reply:
While I appreciate the sentiment outlined here with respect to your inclusion, I would posit that as a community advocate your position on such a team would certainly have very valuable input.

Reply:
I think the RFC still leave open to have a similar arrangement for Nixpkgs/NixOS itself. 
So the current team should be fine.